rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+Exceptions to Scrum (colloquially known as [ScrumButs](https://www.scrum.org/resources/what-scrumbut) in that community) are described below. 
+
+As idenfified by charters, the DCP **Product Owners** (with guidance from **UXR**) are the **!** **Scrum** Product Owners in violation of Scrum’s *"the product owner is one person, not a committee"*.)
+
+There is also no formal DCP **Scrum Master**.
+
+### ZenHub
+
+[ZenHub](https://www.zenhub.com/) has been partially adopted in the DCP community. This proposal commits the community to validating whether ZenHub can become our *one source of truth* for project management. If not, then it should be replaced with a more suitable alternative.
+
+DCP SHOULD NOT duplicate the ZenHub DCP board in other documents or spreadsheets. The board and its reports represent the definitive list of DCP prioritized tasks (Product Backlog) and their state.
+
+### ! Scrum → ZenHub Secret Decoder Ring
+
+* Backlog items → Issues with User Stories
+* Spikes → Issues with a "Spike" GitHub label
+* Epics → Epics
+* Sprint → Milestone
+* Product backlog → Issues without a Milestone in the *Product Backlog* pipeline
+* Sprint Backlog → Issues with a Milestone in the *Sprint Backlog* pipeline
+
+### DCP Product Backlog
+
+The DCP ZenHub board is the ordered **!** **Scrum** Product Backlog for DCP. 
+
+The top-level *dcp* repo MUST contain **all** significant features that are important to the DCP community as a whole and require visibility and prioritization. This includes  issues or epics describing user features or product-wide technical requirements (e.g. “components should have logs...“). 
+
+All component repos that will assign items to a DCP Milestone MUST be added to the ZenHub *DCP Backlogs* workspace.
+
+DCP Product Backlog items  are modeled as:
+- [User Stories](http://agiledictionary.com/277/user-story/) with acceptance criteria
+- [Epics](http://agiledictionary.com/309/epic/) contain a related set of smaller User Stories required to complete an objective.
+- [Spikes](http://agiledictionary.com/209/spike/) adopted from eXtreme Programming to answer design questions.
+  - [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+#### User Stories
+
+The [*researcher with a keyboard*](https://docs.google.com/drawings/d/1Li9o_uVkEZDZHbkQv86I49FG8I0zGSgHYkKabp7piEs/edit) and [*researcher with a pipette*](https://docs.google.com/drawings/d/1o6I5BLtJuRJ1vUdb7dHCkFbo0XiaxKDBiVAogqnUowU/edit) journeys are the initial source of user stories that  are modeled as epics in the *dcp* repo,  reflecting the narrative flow of the product. Further refinement by the Product Owners is required to both develop and  extend the high-level set of user activities. 
+
+Since journeys may vary per persona personas MUST be identified for an epic by assigning a documented GitHub label such as "keyboard", "pipette", or "road-builder".  An epic MAY support multiple personas.
```

Comment:
"road-builder" is not defined (unlike user-with-a-pipette/keyboard) and I'd suggest a less-obfuscating term such as "analysis/external application developer".

Reply:
It was defined in [Focus personas - overview](https://docs.google.com/document/d/1HDV1O2Jl_9LaLAK3zoPb15XSTJb4m0yuPt1oHr2Hs6c/edit) - _The Analysis Road Builder - a developer building analysis tools or portals for others to use_